translations/fr.json

```diff
@@ -258,6 +258,12 @@
         },
         {
           "description": "Write Arabic text on apps that don't support bidirectional text."
+        },
+        {
+          "description": ""
```

Comment:
what's up with this empty description? not that it matters since it's just the french unfinished translation

Reply:
because en.json [showcase.projects] had 4 elements in it and fr.json only 3. To match files i added empty description before adding my text